src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -517,12 +517,13 @@ namespace Akka.Actor
         public static Akka.Actor.DeployableDecider From(Akka.Actor.Directive defaultDirective, System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<System.Type, Akka.Actor.Directive>> pairs) { }
         public static Akka.Actor.LocalOnlyDecider From(System.Func<System.Exception, Akka.Actor.Directive> localOnlyDecider) { }
     }
-    public class DedicatedThreadScheduler : Akka.Actor.SchedulerBase, Akka.Actor.IDateTimeOffsetNowTimeProvider, Akka.Actor.ITimeProvider
+    public class DedicatedThreadScheduler : Akka.Actor.SchedulerBase, Akka.Actor.IDateTimeOffsetNowTimeProvider, Akka.Actor.ITimeProvider, System.IDisposable
```

Comment:
Nah, this is an internal component which *can* be extended but we don't know of any implementations out in the wild. Plus the old API is frankly kind of dangerous (as I explained in my comment below.) 

What I've decided to do was change the `SchedulerBase` constructor that supports the new `Activator.CreateInstance` arguments , but I've left the old `DedicatedThreadScheduler` constructor intact in case anyone was instantiating one of those for some reason. I figured that was a reasonable compromise.